package.json

```diff
@@ -26,19 +26,19 @@
   },
   "homepage": "https://github.com/ethereumjs/ethereumjs-wallet",
   "dependencies": {
-    "aes-js": "^0.2.3",
-    "bs58check": "^1.0.8",
-    "ethereumjs-util": "^4.4.0",
-    "hdkey": "^0.7.0",
+    "aes-js": "^3.1.0",
+    "bs58check": "^2.1.1",
+    "ethereumjs-util": "^5.1.4",
```

Comment:
I wonder if this from ``Wallet.fromV1`` doesn't need to be adopted due to this byte to bit length change in the ``sha3()`` function:

```
var decipher = crypto.createDecipheriv('aes-128-cbc', ethUtil.sha3(derivedKey.slice(0, 16)).slice(0, 16), Buffer.from(json.Crypto.IV, 'hex'))
```

(not possible in GitHub to comment on untouched lines, how stupid is that?)

Reply:
`ethUtil.sha3(derivedKey.slice(0, 16))` just runs a standard 256bit keccak.

Reply:
Ah you mean we may be able to use the 128bits sponge output as it should be identical? I wouldn't do it right now as there is a very sparse test coverage for the V1 wallet (it may also should be removed).

Reply:
I don't understand anything of what you are assuming of what I could mean. 😸 

I actually mis-read the brackets and thought, a ``width`` argument would have been passed to the function, so false alarm.